public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -41,10 +46,16 @@ const ContactPointsTab = () => {
   const { selectedAlertmanager } = useAlertmanager();
   const [queryParams] = useURLSearchParams();
 
+  // If we're using the K8S API, then we don't need to fetch the policies info within the hook,
+  // as we get metadata about this from the API
+  const fetchPolicies = !shouldUseK8sApi(selectedAlertmanager!);
+  // User may have access to list contact points, but not permission to fetch the status endpoint
+  const fetchStatuses = contextSrv.hasPermission(AccessControlAction.AlertingNotificationsRead);
```

Comment:
Without this, a user with basic permissions would end up calling for contact points statuses and get an error toast on screen